simulation/sub8_gazebo/models/sub8/sub8.sdf

```diff
@@ -172,7 +173,14 @@
         </plugin>
       </sensor>
     </link>
+    <!-- Commend out to enable physics simulated controls -->
+    <plugin name="sub8_state_set" filename="libsub8_state_set.so">
+      <staticOffset>0 0 0</staticOffset>
+      <referenceTopic>/c3_trajectory_generator/trajectory_v</referenceTopic>
+      <alwaysOn>true</alwaysOn>
+    </plugin>
 
+    <!-- Uncomment to enable physics simulated controls
```

Comment:
If you want to simulate controls with physics, you comment out the state publisher and uncomment the buoyancy and thruster plugins.